app/components/agenda/agendaitem/agendaitem-controls.js

```diff
@@ -110,4 +119,26 @@ export default class AgendaitemControls extends Component {
   verifyDelete(agendaitem) {
     this.deleteItem(agendaitem);
   }
+  @action
+  async resetDecisionResultCode() {
+    this.isSavingDecisionResult = true;
+    const agendaItemType = await this.args.agendaitem.type;
+    const isAnnouncement =
+      agendaItemType.uri === CONSTANTS.AGENDA_ITEM_TYPES.ANNOUNCEMENT;
+    const defaultDecisionResultCodeUri = isAnnouncement
+      ? CONSTANTS.DECISION_RESULT_CODE_URIS.KENNISNAME
+      : CONSTANTS.DECISION_RESULT_CODE_URIS.GOEDGEKEURD;
+    this.setDecisionResultCode(defaultDecisionResultCodeUri);
+    this.isSavingDecisionResult = false;
+  }
+
+  async setDecisionResultCode(DRC_URI) {
+    // DRC_URI should come from CONSTANTS.DECISION_RESULT_CODE_URIS
```

Comment:
Comment doesn't have a lot of value imo. It has become a commonly used pattern in the codebase so I don't think this needs to be explicated here.

Reply:
passing aliased constants as params in a method might not be so common, was unsure of capitals were better.
Will change and remove comment